README.txt

```diff
@@ -68,6 +68,16 @@ Easy Way
 
     $ pip install gssapi
 
+You can also use a specific Kerberos library by defining the `GSSAPI_*`
+environment variables:
+
+.. code-block:: bash
+
+    $ export GSSAPI_LINKER_ARGS="$(krb5-config --libs gssapi)"
+    $ export GSSAPI_COMPILER_ARGS="$(krb5-config --cflags gssapi)"
+    $ # Too many variants, search for GSSAPI_MAIN_LIB in setup.py
+    $ export GSSAPI_MAIN_LIB="$PATH_TO/lib/yourkerberoslib.ext"
```

Comment:
This should be `PATH_TO/lib/libgssapi.so` with a note saying the extension is `.dylib` on macOS. You might also want to mention that `krb5-config` selected in `PATH` should be for the Kerberos library you wish to link against.

Reply:
There are other usecases on Windows; mingw, MIT Kerberos for Windows etc. Hence the comment. It became too messy to list them all.

Can the comment be improved somehow and still be brief?

Re: PATH, yes. Or maybe specify hard path e.g. `/path/to/you/krb5-config`. What's best?

Reply:
> There are other usecases on Windows; mingw, MIT Kerberos for Windows etc. Hence the comment. It became too messy to list them all.

I can't say I use this with Windows so I can't really speak more, I just think it should be more obvious that the default is `libgssapi.so` for Linux and `libgssapi.dylib` for macOS as per https://github.com/pythongssapi/python-gssapi/blob/a7de108528053417459e6b7ef64aad1b89ab8415/setup.py#L186-L192.

>  Or maybe specify hard path e.g. /path/to/you/krb5-config

You need to make sure it's first in `PATH`. Your linker and compiler args can use the full path but [setup.py calls](https://github.com/pythongssapi/python-gssapi/blob/a7de108528053417459e6b7ef64aad1b89ab8415/setup.py#L129) `krb5-config gssapi --prefix` so you need to ensure the correct `krb5-config` is selected there.

Reply:
> You need to make sure it's first in PATH. Your linker and compiler args can use the full path but setup.py calls krb5-config gssapi --prefix so you need to ensure the correct krb5-config is selected there.

I totally missed that! Fixing.